addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/channels.xml

```diff
@@ -37,5 +38,12 @@
 			<tag>ColorTemperature</tag>
 		</tags>
 	</channel-type>
+    
+    <!-- Command Channel -->
```

Comment:
Please only use spaces for indentation in this file.

Reply:
Tried fixing it, I did use tab, but somehow it reverts back to spaces?

Reply:
Looks to me like it is fixed now.

Reply:
It were spaces when @wborn reviewed, now I can see tabs there. That is why I wrote that it looks fixed to me.

@wborn Just a short ping, maybe you have time to approve this or should we request a review from another dev?